Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN --mount=type=cache,target="/root/.cache/go-build" CGO_ENABLED=0 GOOS=${TARGE
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM alpine:latest
```

Comment:
did this get changed by accident?

Reply:
yeah i was testing for the presence of  config yaml  in the internal folder by executing shell command . Somehow it was missed :(.